model_drafts/tiles/floor.js

```diff
@@ -0,0 +1,42 @@
+var Sequelize = require('sequelize');
+
+function Floor (db) {
+  var Floor = db.define('Floor', {
+    type: Sequelize.TEXT,
+    reference: Sequelize.UUID
+  });
+  return Floor;
+}
+
+// how do we deal with pits? they are a yes/no kind of thing and override everything else.
+
+
+function ConveyorBelt(db){
+  var ConveyorBelt = db.define('ConveyorBelt', {
+    exitto: Sequelize.TEXT,  //n,s,e,w
+    enterfrom: Sequelize.ARRAY(Sequelize.TEXT), //array of one or two letters
+    magnitude: Sequelize.INTEGER, //1,2
+  });
+  return ConveyorBelt;
+}
+
+function Gear(db){
+  var Gear = db.define('Gear', {
+    direction: Sequelize.TEXT //clockwise or counterclockwise
```

Comment:
We could also have direction maybe be a boolean to make it simple where true is clockwise and false is counterclockwise?